apps/sms/views/shared/style/root.css

```diff
@@ -12,7 +12,7 @@ body {
   font-size: 1.6rem;
   line-height: 2.2rem;
   vertical-align: middle;
-  background-color: #FFF;
+  background: var(--header-background);
```

Comment:
I agree we see the full white :) But if we change from "full <something>" to "header <something else> + content", there is still a flickering effect, as compared to "full <header color>" to "header <header color> + content" which is a lot easier to the eyes.

That said, I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1228379 :) Not sure it would be fixed anytime soon...
But see also https://bugzilla.mozilla.org/show_bug.cgi?id=1164123 where we made sure that the header color _is_ the same than the theme color.